src/character/config.rs

```diff
@@ -0,0 +1,49 @@
+use crate::character::class::{FAR_ENEMIES, MEDIUM_ENEMIES, NEAR_ENEMIES};
+use crate::character::Class;
+use crate::location::Distance;
+use serde::{Deserialize, Serialize};
+use std::{fs, path};
+
+#[derive(Debug, Serialize, Clone, Deserialize)]
+pub struct CharacterConfig {
+    pub hero: Class,
+    pub enemies: Vec<Class>,
+}
+
+impl CharacterConfig {
+    pub fn load(file: path::PathBuf) -> Result<Self, Box<dyn std::error::Error>> {
+        let f = fs::File::open(file)?;
```

Comment:
One thing that's kind of important for this feature is that we shouldn't force the user to setup their own config file (we let them do it if they want, but we need to fallback to the default definition).

This means we need to explore how to package the yaml with default classes into the binary. The approach I was thinking of is using [include_bytes](https://doc.rust-lang.org/std/macro.include_bytes.html) or some crate with a similar effect. Then we look for the custom classes file at a known location, e.g. `~/.rpg/config.yml` but if that is missing we use the defaults.